doc/api/v0.yaml

```diff
@@ -56,6 +58,41 @@ paths:
                 $ref: "#/components/schemas/Fund"
         "404":
           description: The requested fund was not found
+    put:
+      operationId: putFund
+      summary: Update or create fund
+      tags: [fund]
+      description: |
+        Update or replace the fund in the db with the one provided.
+      parameters:
+        - in: path
+          name: id
+          schema:
+            type: integer
+          required: true
+      responses:
+        "200":
+          description: Valid response
+        "422":
+          description: The input is malformed.
+
+  /api/v0/funds/:
+    get:
+      operationId: getFunds
```

Comment:
should this return just the ids now?

Reply:
done, I wonder why is the CI not running though